examples/cstg/html/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Standard Integration Example</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+    <link rel="shortcut icon" href="/images/favicon.png" />
+    <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script defer src="{{ UID2_JS_SDK_URL }}"></script>
+    <script>
+      const clientSideIdentityOptions = {
+        subscriptionId: "{{ SUBSCRIPTION_ID }}",
+        serverPublicKey: "{{ SERVER_PUBLIC_KEY }}",
+      };
+
+      let callbackCounter = 0;
+
+      function updateGuiElements(state) {
+        $("#targeted_advertising_ready").text(
+          __uid2.getAdvertisingToken() ? "yes" : "no"
+        );
+        $("#advertising_token").text(String(__uid2.getAdvertisingToken()));
+        $("#login_required").text(
+          __uid2.isLoginRequired() || __uid2.isLoginRequired() === undefined
+            ? "yes"
+            : "no"
+        );
+        $("#update_counter").text(callbackCounter);
+        $("#identity_state").text(String(JSON.stringify(state, null, 2)));
+
+        if (
+          __uid2.isLoginRequired() ||
+          __uid2.isLoginRequired() === undefined
+        ) {
+          $("#login_form").show();
+          $("#logout_form").hide();
+        } else {
+          $("#login_form").hide();
+          $("#logout_form").show();
+        }
+      }
+
+      function onUid2IdentityUpdated(eventType, payload) {
+        if (
+          payload?.identity &&
+          (eventType === "InitCompleted" || eventType === "IdentityUpdated")
+        ) {
+          ++callbackCounter;
+        }
+        updateGuiElements(payload);
+      }
+
+      function onDocumentReady() {
+        $("#logout").click(() => {
+          __uid2.disconnect();
+          updateGuiElements(undefined);
+        });
+
+        $("#login").click(async () => {
+          const email = $("#email").val();
+
+          try {
+            await __uid2.setIdentityFromEmail(email, clientSideIdentityOptions);
+          } catch (e) {
+            console.error("setIdentityFromEmail failed", e);
+          }
+        });
+      }
+
+      window.__uid2 = { callbacks: [] };
```

Comment:
It might work in our case, but I'd rather push the safer pattern in all examples:
```window.__uid2 = window.__uid2 || {};
window.__uid2.callbacks = window.__uid2.callbacks || [];```

Reply:
How about the slightly shorter

```
window.__uid2 = window.__uid2 || { callbacks: [] };
```

like here:
https://github.com/IABTechLab/uid2-web-integrations/pull/4/files#diff-4ee0cf5a42db1ffaddca2885006d61b391775912c188e9ccc009bfdc39566cbaR19